odoo/addons/base/i18n/it.po

```diff
@@ -17594,7 +17594,7 @@ msgstr "Signor"
 #: model:ir.module.module,shortdesc:base.module_web_mobile
 #: model_terms:ir.ui.view,arch_db:base.contact
 msgid "Mobile"
-msgstr "Dispositivo mobile"
+msgstr "Cellulare"
```

Comment:
In Italian it's way better to say "Cellphone" than "Mobile device" as "device" is a formal word not colloquial at all, that sounds good in mechanic device" or a "medical device" and not something you have in your hands everyday.